react-components/src/utils/authUtils.js

```diff
@@ -15,9 +15,29 @@
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 import { ApolloLink } from '@apollo/client';
 import { checkCookie, cookieValue } from './cookieUtils';
+import { checkItem, getJsonItem } from './localStorageUtils';
 
-const authLink = new ApolloLink((operation, forward) => {
+const SIGN_IN_TOKEN_KEY = 'M2_VENIA_BROWSER_PERSISTENCE__signin_token';
+
+const getAuthToken = () => {
     let token = checkCookie('cif.userToken') ? cookieValue('cif.userToken') : '';
+    if (checkItem(SIGN_IN_TOKEN_KEY)) {
```

Comment:
I guess it would be better to accept the item key as parameter of `getAuthToken()` itself. 

The name suggests that it can be different for different ui implementations.

Reply:
https://github.com/magento/pwa-studio/search?q=signin_token